test/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "bedrock-payment-paypal-test-project",
+  "version": "0.0.1",
+  "description": "Integration Tests for Bedrock Payment PayPal.",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+    "test": "node --preserve-symlinks test test"
+  },
+  "keywords": [
+    "test",
+    "bedrock-payment",
+    "bedrock-payment-paypal"
+  ],
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "https://digitalbazaar.com"
+  },
+  "license": "BSD-3-Clause",
+  "dependencies": {
+    "bedrock": "^3.0.1",
+    "bedrock-injector": "^1.0.0",
```

Comment:
Doesn't need to be addressed in this PR, but I noticed elsewhere that bedrock-injector was superceeded by bedrock-package-manager.